src/fights.ts

```diff
@@ -667,8 +669,12 @@ const pygmySniffed = () =>
   sniffSources.some((source) => pygmyBanishHandlers.some(({ pygmy }) => pygmy === get(source)));
 
 const pygmyMacro = Macro.step(
-  ...pygmyBanishHandlers.map(({ pygmy, skill, item }) =>
-    Macro.if_(pygmy, skill ? Macro.trySkill(skill).item(item) : Macro.item(item)),
+  ...pygmyBanishHandlers.map(({ pygmy, skill, item, check, limit }) =>
```

Comment:
god this is so many vars this isn't a review comment just jesus christ i forgot how terrible pygmies are